release-notes.txt

```diff
@@ -1,3 +1,550 @@
+RIOT-2019.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.07 release includes a number of new features including many new
+boards and cpu, riotboot added to many new and old boards, usb is now available,
+ble improvements, ethernet on stm32 platforms, progress in gnrf fragmentation,
+as well as many bug fixed and documentation updates.  Testing has also improved
+with both On-Target testing increasing and now Hardware Assisted automated tests
+being run.
+
+About 300 pull requests with about 659 commits have been merged since the last
+release and about 50 issues have been solved. 26 people contributed with code
+in 106 days. Approximately 1377 files have been touched with 181993 insertions
+and 19668 deletions.
+
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
```

Comment:
Many things below are rather bug fixes...

Reply:
bug fix or change?

Reply:
For bug fixes we have the "issues fixed" section. Non-critical issues that came up during the release cycle are usually not worth mentioning.